addons/purchase/i18n/purchase.pot

```diff
@@ -37,7 +37,7 @@ msgstr ""
 #. module: purchase
 #. odoo-python
 #: code:addons/purchase/models/purchase_order.py:0
-msgid "%(amount)s due %(date)s"
+msgid "%(amount)s, Order due %(date)s"
```

Comment:
Since this is done in master, there's no need to modify the pot file. It is re-generated after each freeze.